components/Search.js

```diff
@@ -45,6 +46,7 @@ class Search extends Component {
       },
       () => this.input.focus()
     );
+    this.props.onChange(evt);
```

Comment:
So slotting this in here is actually running it twice for people who actually use the onChange input (once on the section where we wipe out the value, again here). I think what should happen is that you should merge the two objects - `evt` and then the faux `evt` object I have up there and then run the this.props.onchange once. You'd have to reverse the logic up there as well. It'd look something like

```js
if (!this.props.value) {
   this.input.value
} 

...

const newEvt = Object.assign(evt, { target: { value: '' } });
this.props.onChange(newEvt)
```